<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="PO-3296-5MSY1" name="com.hbt.semillero.servicios.ejb.GestionarSaldosTest" tests="6" failures="0" timestamp="06 nov 2019 20:25:53 GMT" time="0.022" errors="1">
  <testcase name="eliminarSaldoTest" time="0.001" classname="com.hbt.semillero.servicios.ejb.GestionarSaldosTest"/>
  <testcase name="recorrerSaldo" time="0.000" classname="com.hbt.semillero.servicios.ejb.GestionarSaldosTest"/>
  <testcase name="consultarSaldo" time="0.019" classname="com.hbt.semillero.servicios.ejb.GestionarSaldosTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.hbt.semillero.servicios.ejb.GestionarSaldos.consultarSaldo(GestionarSaldos.java:73)
at com.hbt.semillero.servicios.ejb.GestionarSaldosTest.consultarSaldo(GestionarSaldosTest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)
]]>
    </error>
  </testcase> <!-- consultarSaldo -->
  <testcase name="crearSaldoTest" time="0.001" classname="com.hbt.semillero.servicios.ejb.GestionarSaldosTest"/>
  <testcase name="crearSaldosTest" time="0.001" classname="com.hbt.semillero.servicios.ejb.GestionarSaldosTest"/>
  <testcase name="modificarSaldoTest" time="0.000" classname="com.hbt.semillero.servicios.ejb.GestionarSaldosTest"/>
</testsuite> <!-- com.hbt.semillero.servicios.ejb.GestionarSaldosTest -->
